compilation error for matlab bindings (mex)

I get two errors when compiling the matlab bindings for **tixi-2.2.4** on
**Linux Mint 18.1 Cinnamon 64-bit**
using the
**gcc version 5.4.0**

tixi/matlab/common.h38:1: error: C++ style commens are not allowed in ISO C90
tixi/matlab/tiximatlab.c:4035:9: error: C++ style commens are not allowed in ISO C90

![grafik](https://user-images.githubusercontent.com/10432668/27220711-6be4787a-5286-11e7-9213-3e5f1db3db9a.png)

![grafik](https://user-images.githubusercontent.com/10432668/27220944-21f981d2-5287-11e7-9b72-31a6395dcd60.png)

Can be fixed by changing the comment style from // to /**/

--- a/bindings/bindings_generator/matlab_generator.py
+++ b/bindings/bindings_generator/matlab_generator.py
@@ -402,7 +402,7 @@ extern "C" {
             call += '}\n'
         
         call += '''else {
-    // this is only executed if the function could not be identified
+    /* this is only executed if the function could not be identified */
     char text[255];
 '''
         call += '    snprintf(text, 250, "%%s is not a valid %s function!\\n", functionName);\n' % self.prefix

--- a/bindings/matlab/common.h
+++ b/bindings/matlab/common.h
@@ -35,7 +35,7 @@
 #include "mex.h"
 
 #ifndef __PACKAGENAME__
-// define a generic package name if macro is not defined in including source file
+/* define a generic package name if macro is not defined in including source file */
 #define __PACKAGENAME__ "TIXI"
 #endif
 

